README.md

```diff
@@ -1,9 +1,9 @@
 # jv-oop
 
-- Create class Machine containing methods `public abstract void doWork()` and `public abstract void stopWork()`.  
-- Create classes `Truck`, `Bulldozer`, and `Excavator` that will inherit from `Machine`.
+- Create class core.machines.Machine containing methods `public abstract void doWork()` and `public abstract void stopWork()`.  
+- Create classes `core.machines.Truck`, `core.machines.Bulldozer`, and `core.machines.Excavator` that will inherit from `core.machines.Machine`.
```

Comment:
be careful with project-wide renamings, moving files - review your files before committing to avoid unintended changes, please undo the change